airflow/api_connexion/openapi/v1.yaml

```diff
@@ -377,6 +377,12 @@ paths:
       parameters:
         - $ref: '#/components/parameters/PageLimit'
         - $ref: '#/components/parameters/PageOffset'
+       - name: only_active
```

Comment:
```suggestion
        - name: only_active
```

Reply:
You're right - but I also missed adding the new fields in `test_dag_schema` when I originally opened this PR.